test-framework/oauth/src/main/java/org/keycloak/testframework/oauth/OAuthClientSupplier.java

```diff
@@ -43,7 +44,11 @@ public OAuthClient getValue(InstanceContext<OAuthClient, InjectOAuthClient> inst
         String clientId = testAppClient.getClientId();
         String clientSecret = testAppClient.getSecret();
 
-        ApiUtil.handleCreatedResponse(realm.admin().clients().create(testAppClient));
+        try (Response response = realm.admin().clients().create(testAppClient)) {
+            if (!Response.Status.CONFLICT.equals(Response.Status.fromStatusCode(response.getStatus()))) {
+                ApiUtil.handleCreatedResponse(response);
+            }
+        }
```

Comment:
This doesn't make much sense. I'd say we should update `ApiUtil` to handle this case.

Reply:
ok, by me :+1: 

This time it was about OAuth client inject has been trying to create the client repeatedly.